.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
```

Comment:
- [ ] Great job installing the linters! 💯 However, there's a small mistake with them: You named the folder container as `".github/workflows"` and the correct way should be one folder called `".github"` that contains another folder called `"workflows"` that contains the `"linters.yml"` file. Kindly consider fixing this issue! 🤓 🛠️

Reply:
Hello Andres,
Thank You for being kind to me. It seems that the directory changes on github because everything is intact in my local repo. 
Rather, Github displays my directory as a single directory. Do you think that it is because .github folder is empty? 

I will attach a snapshot to show my local repo 

![local repo](https://user-images.githubusercontent.com/117144028/204637100-83e49d49-511d-4501-812c-a8ad353e2156.JPG)